closed/custom/Docs.gmk

```diff
@@ -259,9 +253,7 @@ OPENJ9_ONLY_JAVADOC_SCRIPT_BODY = \
       regex="$$topdir/.*$$excepttopdir" ; \
       if ! $(GREP) -qe "$$regex" $(OPENJ9_ONLY_JAVADOC_FILES) ; then \
```

Comment:
What is the point of this use of `grep`? The files named in `OPENJ9_ONLY_JAVADOC_FILES` all match `*.java` and the files we're considering here are named `module-summary.html`.

Reply:
The code is looking for a module-info.java file provided by openj9 - in that case we want to leave the generated module-summary.html unchanged.
So for example:
1. When the regex is  openj9.cuda/.*module-info.java a file is found and the module-summary.html is left unchanged.
2. When the regex is  jdk.management/.*module-info.java no file is found and the module-summary.html is edited.